app/api/migrations/migrations/34-move-attachments/index.js

```diff
@@ -15,14 +15,15 @@ export default {
       const entity = await cursor.next();
 
       await Promise.all(
-        entity.attachments.map(async ({ filename, originalname }) =>
+        entity.attachments.map(async ({ filename, originalname, mimetype }) =>
```

Comment:
@konzz Bonus comment: this change makes sense as long as we don't deploy the PR in which this migration was included. Did we deploy it already?

Reply:
Yes, when I did it, it was not deployed yet, I removed it since it will need to go in a new migration anyway